DOCSP-46973-adds-large-destination-index-FAQ (#579)
mongodb/docs-cluster-to-cluster-sync: source/faq.txt

   Index builds on the destination cluster are treated as writes 
   while ``mongosync`` is syncing.

Why are the destination cluster indexes larger than the source cluster indexes?
-------------------------------------------------------------------------------

The following factors may contribute to an increase in index size on destination
clusters:

- ``mongosync`` inserts and removes data during a migration, which may cause disk 

>   fragmentation.

Reviewer: instead of disk fragmentation, can we say that it can cause data to be stored inefficiently on disk? I checked with the server team, and they noted that disk fragmentation isn't the best description

Author: Updated!

- By default, ``mongosync`` builds indexes before copying data. ``mongosync`` 

>   copies data in ``_id`` order. If the index is not correlated with  ``_id``, 
>   the index size can become large. For more information, see the MongoDB Manual

Reviewer 2: [suggestion] for clarity

```suggestion
  copies data in ``_id`` order. If an index is not correlated with  ``_id``, 
  the index size can become large. For more information, see the MongoDB Manual
```

  :ref:`FAQ: Indexes<faq-indexes-random-data-performance>` page.

Use the following methods to mitigate an increase in index size:

- Restart the migration with the ``buildIndexes`` 

>   :ref:`parameter <c2c-api-start-params>` set to ``never``. When the migration 
>   is complete, manually build indexes on the destination cluster.

Reviewer 2: [suggestion] for minimalism

```suggestion
  :ref:`parameter <c2c-api-start-params>` set to ``never``. When the migration 
  finishes, manually build indexes on the destination cluster.
```

> - After the migration, perform a rolling sync on the destination cluster.

Reviewer: ```suggestion
```
We're not 100% confident on this suggestion so I think we can omit it here.

> - After the migration, perform a rolling sync on the destination cluster.

Reviewer: ```suggestion
- After the migration, perform a rolling initial sync on the destination cluster.
```

we could also link these docs: https://www.mongodb.com/docs/manual/core/replica-set-sync/#replica-set-data-synchronization

Author: Updated this sentence to say:

After the migration, perform a rolling [initial sync](https://www.mongodb.com/docs/manual/core/replica-set-sync/#replica-set-data-synchronization) on the destination cluster.

> - After the migration, run :ref:`<compact>` on the destination cluster. This 

Reviewer: can we also note that this operation is slow?

Reviewer 3: alternatively, should we link to https://www.mongodb.com/docs/manual/reference/command/compact/#performance-considerations on compact, to avoid potential duplication of context / notes on compact that can get out of sync with the main compact docs?

Reviewer: agreed

Author: I rewrote the last sentence to say:

This rebuilds indexes and releases unneeded disk space to the OS, but may impact cluster [performance](https://www.mongodb.com/docs/manual/reference/command/compact/#performance-considerations).
